_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1027,28 +1121,40 @@ repo, setting the `ref` param to the version you require:
 [source,hcl]
 ----
 terraform {
-  source = "git::git@github.com/gruntwork-io/terraform-aws-cis-service-catalog//modules/landingzone/account-baseline-root?ref=v0.22.0"
-
-  # This module deploys some resources (e.g., AWS Config) across all AWS regions, each of which needs its own provider,
-  # which in Terraform means a separate process. To avoid all these processes thrashing the CPU, which leads to network
-  # connectivity issues, we limit the parallelism here.
-  extra_arguments "parallelism" {
-    commands  = get_terraform_commands_that_need_parallelism()
-    arguments = ["-parallelism=2"]
-  }
+  source = "git::git@github.com:gruntwork-io/terraform-aws-cis-service-catalog.git//modules/landingzone/account-baseline-root?ref=v0.27.0"
 }
 ----
 
-[.exceptional]
-IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=2`), as shown above, with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors.
-
 Set the variables for the `account-baseline-root` module in this environment in the `inputs = { ... }` block of `terragrunt.hcl`:
 
 .infrastructure-live/root/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
-locals {
+# ---------------------------------------------------------------------------------------------------------------------
+# CONFIGURE A PROVIDER FOR EACH AWS REGION
+# To deploy a multi-region module, we have to configure a provider with a unique alias for each of the regions AWS
+# supports and pass all these providers to the multi-region module in a provider = { ... } block. You MUST create a
+# provider block for EVERY one of these AWS regions, but you should specify the ones to use and authenticate to (the
+# ones actually enabled in your AWS account) using opt_in_regions.
+# ---------------------------------------------------------------------------------------------------------------------
 
+generate "providers" {
```

Comment:
SUPER NIT: I found it slightly confusing to understand the `generate` block here, as it references `locals` defined below it. We typically put the `locals` at the top for this reason. This is probably as low priority of a NIT as exists in the world, so it's totally OK to ignore this entirely 😁

Reply:
Addressed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/647/commits/ca08c5c99ae6af530a0766cb017f29afb35a32cf.